qiskit_nature/second_q/circuit/library/ansatzes/ucc.py

```diff
@@ -310,7 +311,12 @@ def operators(self):  # pylint: disable=invalid-overridden-method
                 # inserting ``None`` as the result if an operator did not commute. To ensure that
                 # the ``excitation_list`` is transformed identically to the operators, we retain
                 # ``None`` for non-commuting operators in order to manually remove them in unison.
-                operators = self.qubit_converter.convert_match(excitation_ops, suppress_none=False)
+                if isinstance(self.qubit_converter, QubitConverter):
+                    operators = self.qubit_converter.convert_match(
+                        excitation_ops, suppress_none=False
+                    )
+                else:
+                    operators = self.qubit_converter.map(excitation_ops)
```

Comment:
We should add a comment here, again referring to https://github.com/Qiskit/qiskit-nature/issues/974.
Once we work on that mapper we need to think about how to handle the `suppress_none=False` case
which we had in the `QubitConverter`.